guides/source/active_storage_overview.md

```diff
@@ -129,9 +129,12 @@ amazon:
   secret_access_key: ""
   region: ""
   bucket: ""
+  http_read_timeout: "" # default: 60
+  retry_limit: "" # default: 3
   upload: 
     server_side_encryption: "" # 'aws:kms' or 'AES256'
 ```
+NOTE: Set sensible client http timeouts and retry limits for your application. Using the default configuration, connections may be held for up to 240 seconds in certain error scenarios, and lead to request queuing.
```

Comment:
Since this may change in the future, what if we avoided specifics?

```suggestion
TIP: Set sensible client HTTP timeouts and retry limits for your application. In certain failure scenarios, the default AWS client configuration may cause connections to be held for up to several minutes and lead to request queuing.
```

Reply:
good call, i like the language you suggested